docs/tutorials/index.md

```diff
@@ -22,9 +22,9 @@ title: "Quickstart (local)"
   ~ under the License.
   -->
 
-This quickstart gets you started with Apache Druid and introduces you to Druid ingestion and query features. For this tutorial, we recommend a machine with at least 6 GB of RAM.
+This quickstart helps you install Apache Druid and introduces you to Druid ingestion and query features. For this tutorial, you need a machine with at least 16 GiB of RAM.
```

Comment:
Just checking whether 6 GB should be changed to 16 GiB?

Reply:
@vtlim changed this based upon the following from #Requirements (which should be pre-requisites)
> You can follow these steps on a relatively modest machine, such as a workstation or virtual server with 16 GiB of RAM.

Reply:
@findingrish can you help clarify here? thanks!

Reply:
The script requires a minimum memory of 4250 MB (when starting all services). 

We can use a slightly higher number in the docs, 5GB or 6GB.